stories/components/FileCollection/FileCollection.md

```diff
@@ -0,0 +1,109 @@
+# FileCollection
+
+**Available components**:
+
+- [FileCollection](#filecollection)
+
+**How it is built**
+
+- uses react-dropzone library
+
+## KEY CONSIDERATIONS WHEN USING COMPONENT
+
+### UploadedFiles prop
+
+UploadedFiles prop is an array that can be passed as a json object. A key of 'keep' is mandatory to be passed in as this is what keeps track of whether or not the file should be kept or removed/deleted. If an uploaded file is removed/deleted through ui, `uploadedFile.keep = false`. If an uploaded file is not removed, `uploadedFile.keep = true`
+
+Example:
+
+```javascript
+  {
+    name: 'file1.txt',
+    src: 'https://www.google.ca',
+    file_type: 'text/plain',
+    keep: true,
+  },
+```
+
+### Handling Errors
+
+Error is not handled in the component. This comes from the fact that we are tracking whether or not an uploaded file should be kept or deleted.
+Error can be handled by checking if `newFile[].length > 0 || !values.uploadedFiles.every(file => file.keep === false)`
```

Comment:
This wording is pretty confusing
Also this double negative `!values.uploadedFiles.every(file => file.keep === false)`
Could this be a .some?